src/app/themes/volumio/components/track-manager/volumio-track-manager.html

```diff
@@ -39,9 +39,21 @@
         <div class="smallText">
           {{trackManager.playerService.songLength}}
         </div>
-        <div class="footerText">
-          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'">
-            {{trackManager.playerService.state.trackType}}
+        <div class="footerText" title="Audio Format">
+          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'" ng-switch on="trackManager.playerService.state.trackType">
```

Comment:
See.. I knew you would know there was a better way of doing it! Thanks for the tip.